dataline-workers/src/test/resources/simple_postgres_init.sql

```diff
@@ -1,12 +1,33 @@
-CREATE TABLE id_and_name
-(
-    id   integer,
-    name VARCHAR(200)
-);
+CREATE
+    TABLE
+        id_and_name(
+            id INTEGER,
+            name VARCHAR(200)
+        );
 
-INSERT INTO id_and_name (id, name)
-VALUES (1, 'sherif'),
-       (2, 'charles'),
-       (3, 'jared'),
-       (4, 'michel'),
-       (5, 'john');
+INSERT
+    INTO
+        id_and_name(
+            id,
+            name
+        )
+    VALUES(
```

Comment:
this feels overly aggressive in line breaking SQL statements. WDYT? Can we make it more tolerate e.g: 150 chars line width?

Reply:
I don't have control on this unfortunately. If it is too ugly, I can disable SQL formatting.

Reply:
not a dealbreaker for me but I prefer disabling